packages/plugin-preload/src/index.ts

```diff
@@ -1,365 +1,377 @@
+      type: ParameterType.TIMELINE,
+      pretty_name: "Trials",
       default: [],
-      description:
-        "Array with a timeline of trials to automatically preload. If one or more trial objects is provided, " +
-        "then the plugin will attempt to preload the media files used in the trial(s).",
     },
+    /* Array with one or more image files to load. This parameter is often used in cases where media files cannot
+    be automatically preloaded based on the timeline, e.g. because the media files are passed into an image plugin/parameter with 
+    timeline variables or dynamic parameters, or because the image is embedded in an HTML string. */
     images: {
-      type: jsPsych.plugins.parameterType.STRING,
+      type: ParameterType.STRING,
+      pretty_name: "Images",
       default: [],
-      description:
-        "Array with one or more image files to load. This parameter is often used in cases where media files cannot " +
-        "be automatically preloaded based on the timeline, e.g. because the media files are passed into an image plugin/parameter with " +
-        "timeline variables or dynamic parameters, or because the image is embedded in an HTML string.",
     },
+    /* Array with one or more audio files to load. This parameter is often used in cases where media files cannot 
+    be automatically preloaded based on the timeline, e.g. because the media files are passed into an audio plugin/parameter with 
+    timeline variables or dynamic parameters, or because the audio is embedded in an HTML string. */
     audio: {
-      type: jsPsych.plugins.parameterType.STRING,
+      type: ParameterType.STRING,
+      pretty_name: "Audio",
       default: [],
-      description:
-        "Array with one or more audio files to load. This parameter is often used in cases where media files cannot " +
-        "be automatically preloaded based on the timeline, e.g. because the media files are passed into an audio plugin/parameter with " +
-        "timeline variables or dynamic parameters, or because the audio is embedded in an HTML string.",
     },
+    /* Array with one or more video files to load. This parameter is often used in cases where media files cannot 
+    be automatically preloaded based on the timeline, e.g. because the media files are passed into a video plugin/parameter with 
+    timeline variables or dynamic parameters, or because the video is embedded in an HTML string. */
     video: {
-      type: jsPsych.plugins.parameterType.STRING,
+      type: ParameterType.STRING,
+      pretty_name: "Video",
       default: [],
-      description:
-        "Array with one or more video files to load. This parameter is often used in cases where media files cannot " +
-        "be automatically preloaded based on the timeline, e.g. because the media files are passed into a video plugin/parameter with " +
-        "timeline variables or dynamic parameters, or because the video is embedded in an HTML string.",
     },
```

Comment:
Yes! I added `array: true` to `audio`, `images` and `video`. But I didn't add it to `trials` because that has type `TIMELINE`, which I think has to be an array. So adding `array: true` here would mean an array of timeline arrays (similar to not using `array: true` with the type `KEYS`). Is that right?

Reply:
Absolutely right! I misread the description. Maybe "A timeline of trials to automatically preload" would be a bit clearer.

Reply:
Agree. Done!